### ISSUE TITLE
[REACT] react click outside of component 

### ISSUE THREAD

**Issue by quasarchimaere** [researchstudio-sat/webofneeds#3030]
...is not handled (find `//TODO: REACT ON CLICK OUTSIDE OF COMPONENT` to see where it needs to be implemented)

Needed so dropdowns close if there is a click outside of the component

**Comment by quasarchimaere** [comment]
fixed in the refac__ng_to_react branch, no longer relevant once the branch is done and meeged
